bobtimus/src/http.rs

```diff
@@ -24,11 +24,14 @@ where
     R: RngCore + CryptoRng + Clone + Send + Sync + 'static,
     RS: LatestRate + Clone + Send + Sync + 'static,
 {
-    let index_html = warp::path::end().and_then(serve_index);
-    let waves_resources = warp::path::tail().and_then(serve_waves_resources);
+    let index_html = warp::get().and(warp::path::end()).and_then(serve_index);
```

Comment:
:question: Was the `GET` implicit before?

Reply:
I guess so :)